lib/elastic_apm.rb

```diff
@@ -361,7 +361,7 @@ def report_message(message, context: nil, **attrs)
     # Set a _tag_ value for the current transaction
     #
     # @param key [String,Symbol] A key
-    # @param value [Object] A value (will be converted to string)
```

Comment:
should we perhaps leave `set_tag` alone, and make this change on `set_label` in #543? `set_tag` would call `set_label(key, value.to_s)`?

Reply:
I like this idea.

Reply:
Yes, this is great! That way, it won't be a breaking change to allow boolean and numerical tag values. Thanks for the suggestion @axw